java-library/src/test/java/uk/co/lucasweb/aws/v4/signer/Sha256Test.java

```diff
@@ -36,7 +35,9 @@
 
     @Test
     public void shouldGetSha256() {
-        assertThat(Sha256.get(TEST, Charset.forName("UTF-8")))
+        String contentHash = Sha256.INSTANCE.digestAsHexString(TEST);
+
+        assertThat(contentHash)
```

Comment:
We're relying on the existing tests to verify the kotlin SHA-256 implementation behaves the same as the java one.